src/Action/DeleteProjection.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * This file is part of the prooph/event-store-http-middleware.
+ * (c) 2016-2018 prooph software GmbH <contact@prooph.de>
+ * (c) 2016-2018 Sascha-Oliver Prolic <saschaprolic@googlemail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Prooph\EventStore\Http\Middleware\Action;
+
+use Prooph\EventStore\Exception\ProjectionNotFound;
+use Prooph\EventStore\Projection\ProjectionManager;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\ServerRequestInterface;
+use Psr\Http\Server\RequestHandlerInterface;
+
+final class DeleteProjection implements RequestHandlerInterface
+{
+    /**
+     * @var ProjectionManager
+     */
+    private $projectionManager;
+
+    /**
+     * @var ResponseInterface
+     */
+    private $responsePrototype;
```

Comment:
sorry, I don't get it, why response prototype instead of simply response?

Reply:
because the package does not depend on diactoros ;)
See https://www.php-fig.org/psr/psr-15/#13-generating-responses

> t is RECOMMENDED that any middleware or request handler that generates a response
will either compose a prototype of a PSR-7 ResponseInterface or a factory capable
of generating a ResponseInterface instance in order to prevent dependence on a
specific HTTP message implementation.